cg/meta/qc_metrics/collect_qc_metrics.py

```diff
@@ -0,0 +1,85 @@
+"""Module for the collect qc metrics meta api."""
+
+import logging
+from typing import Iterator
+
+from housekeeper.store.models import File, Tag
+
+from cg.clients.arnold.api import ArnoldAPIClient
+from cg.clients.janus.api import JanusAPIClient
+from cg.clients.janus.dto.create_qc_metrics_request import (
+    CreateQCMetricsRequest,
+    FilePathAndTag,
+)
+from cg.clients.janus.exceptions import JanusClientError, JanusServerError
+from cg.exc import HousekeeperFileMissingError
+from cg.meta.meta import MetaAPI
+from cg.models.cg_config import CGConfig
+from cg.store.models import Case, Sample
+
+LOG = logging.getLogger(__name__)
+
+
+class CollectQCMetricsAPI(MetaAPI):
+    """Collect qc metrics API."""
+
+    def __init__(self, config: CGConfig):
+        super().__init__(config)
+        self.janus_api: JanusAPIClient = config.janus_api
+        self.arnold_api: ArnoldAPIClient = config.arnold_api
+
+    def get_qc_metrics_file_paths_for_case(self, case_id: str) -> list[File]:
+        """Get the multiqc metrics files for a case."""
+        files: list[File] = self.housekeeper_api.get_files(
+            bundle=case_id, tags=["qc-metrics", "janus"]
+        ).all()
+        if not files:
+            raise HousekeeperFileMissingError(f"No qc metrics files found for {case_id}.")
+        return files
+
+    @staticmethod
+    def get_tag_to_include(tags: list[Tag]) -> str:
```

Comment:
Slight preference for "get_tag_name_to_include" but feel free to keep as is.

Reply:
The tag is the tag name in itself, just the way how it is retrieved from HK it comes with a bunch of extra info. Will keep it as tag. since that is what we want ultimately